tests/test_octodns_provider_ns1.py

```diff
@@ -1123,6 +1094,7 @@ def test_data_for_dynamic_A(self):
         # Test out a small, but realistic setup that covers all the options
         # We have country and region in the test config
         filters = provider._get_updated_filter_chain(True, True)
+        catchall_pool_name = '{}{}'.format(provider.CATCHALL_PREFIX, 'iad')
```

Comment:
Made this change since, the last two times, tests failed due to the changing prefix in the code while the pool name was hardcoded here.